src/auro-datepicker.js

```diff
@@ -784,19 +727,18 @@ export class AuroDatePicker extends LitElement {
           this.inputList[1].value = '';
         }
       }
+
+      this.validation.validate(this);
     }
 
     if (changedProperties.has('error')) {
+      const lastInput = this.inputList[this.inputList.length - 1];
```

Comment:
We need more comments to help understand what is going on in code.

I only understand this because we discussed it.

There should be comments that explain that "error" attributes are passed down to the input and that in the case of `range` supported datepicker, only to the second input. And that this is to prevent the range supported datepicker from displaying the error icon in both inputs.

Reply:
Addressed